src/Docker.DotNet/DockerClient.cs

```diff
@@ -35,8 +35,10 @@ public sealed class DockerClient : IDockerClient
 
         public ISwarmOperations Swarm { get; }
 
+        public TimeSpan DefaultTimeout { get { return _defaultTimeout; } set { _defaultTimeout = value; } }
```

Comment:
Done in latest push. Also merged upstream and made sure the HTTP methods were correct from previous pull request.